modules/users/client/views/settings/manage-social-accounts.client.view.html

```diff
@@ -8,8 +8,11 @@ <h3 class="col-md-12 text-center" ng-show="vm.hasConnectedAdditionalSocialAccoun
     </div>
     <div ng-repeat="(providerName, providerData) in vm.user.additionalProvidersData" class="social-account-container">
       <img ng-src="/modules/users/client/img/buttons/{{providerName}}.png">
-      <a class="btn btn-danger btn-add-remove-account" ng-click="vm.removeUserSocialAccount(providerName)">
-        <i class="glyphicon glyphicon-trash"></i>
+      <a ng-click="vm.removeUserSocialAccount(providerName)">
+        <img class="social-button" ng-src="/modules/users/client/img/buttons/{{providerName}}.png">
```

Comment:
These social provider images are being shown twice. If you see above at line 10, the `<img>` is present there as well. Remove whichever one we don't need. I'm guessing we want this new one that you added.